packaging/windows/Jenkinsfile.groovy

```diff
@@ -0,0 +1,72 @@
+def getCommit(path) {
+    dir(path) {
+        return bat(returnStdout: true, script: '@echo off && git rev-parse HEAD').trim()
+    }
+}
+
+def doBuild(){
+  stage 'Checkout Client'
+  bat 'if EXIST src\\github.com\\keybase\\client\\shared cd src\\github.com\\keybase\\client && git checkout shared'
+  bat 'if EXIST src\\github.com\\keybase\\client\\desktop\\shared cd src\\github.com\\keybase\\client && git checkout desktop/shared'
```

Comment:
symlinks, it is a nightmare at the junction of Windows/Node/Git. I'll make a comment.